### DISCUSSION TITLE
Timer automatically restarts

### DISCUSSION THREAD

**Discussion by Dix1110** [YetAnotherPomodoroApp/YAPA-2#123] [Ideas]
First, thank you for an amazing app.
Now, when I finish a timer, I want the timer to reset when I hit the stop button.
What happens by using all the available options is that:
either the break will start or the next timer will start.
I don't want anything to start but to reset the timer completely to the original set time.
Thank you in advance for your help.

**Comment by SwapnilChand** [comment]
Timer does reset when you hit the stop button, if you do so in the middle of a session. If you finish a session the next timer will start. I think it works fine, or maybe couldn't get you
